README.md

```diff
@@ -1 +1,5 @@
 # ecommerce-test
+#problems i am still figuring out
+- the search filter:i got the filter to identify items with the same string as the input.value but i dont know how to display the search result
```

Comment:
place these comments into your issues. create a new issue for each.

Reply:
sorry i am just seeing these messages.... i'd do that